### PR TITLE
Add linux aarch64

### PR DESCRIPTION
**Pull request by rfiorella**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by rfiorella** [comment]
Removed OSX arm64 since this has already been started as part of #23

**Comment by rfiorella** [comment]
@conda-forge-admin, please restart ci

Old builds had hung on missing aarch64 version of serf; this has been resolved now.

**Comment by rfiorella** [comment]
@zklaus Could you take a look at this PR? Happy to make changes as needed.

File: recipe/meta.yaml [hunk 1]
```diff
@@ -10,12 +10,12 @@ source:
   sha256: 949efd451a09435f7e8573574c71c7b71b194d844890fa49cd61d2262ea1a440
 
 build:
-  number: 0
+  number: 1
   skip: true  # [win]
 
 requirements:
   build:
-    - {{ compiler('c') }}
```
Comment (zklaus) [inline recipe/meta.yaml:18]: We should *add* stdlib in addition to the c compiler, not replace it.

**Review by zklaus — CHANGES_REQUESTED** [review]
Thanks for working on this, @rfiorella! Looks good to me except for one small comment.

File: recipe/meta.yaml (hunk 1 above)
Comment (rfiorella) [reply recipe/meta.yaml:18]: Done with 442735c. 

Thanks! I see now that I had misread the guidance around this change.

**Comment by zklaus** [comment]
@conda-forge-admin, please rerender

**Pull request merged**